### DISCUSSION TITLE
Q: 为什么用流量可以打开页面用wifi打不开啊T T || Q: Why can I open the page using traffic but not using wifi? T T

### DISCUSSION THREAD

**Discussion by starletaxx** [walinejs/waline#2206] [Q&A]
### Describe the bug | 问题描述

我用流量可以打开，用wifi打不开页面，发现是因为用wifi打不开https://unpkg.com/

### Website URL | 问题网站

vercel.hopeandsun.cn

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

I can open the page using data traffic, but cannot open the page using wifi. I found out that it is because https://unpkg.com/ cannot be opened using wifi.

### Website URL | Problem website

vercel.hopeandsun.cn

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
unpkg 的网络问题我们无法解决，你可以选择其他 CDN

---
We cannot solve the network problem of unpkg. You can choose other CDN
